go-controller/pkg/config/config.go

```diff
@@ -2311,10 +2320,11 @@ func buildOvnKubeNodeConfig(ctx *cli.Context, cli, file *config) error {
 	if OvnKubeNode.Mode != types.NodeModeFull && HybridOverlay.Enabled {
 		return fmt.Errorf("hybrid overlay is not supported with ovnkube-node mode %s", OvnKubeNode.Mode)
 	}
+
 	// when DPU is used, management port is backed by a VF. get management port VF information
 	if OvnKubeNode.Mode == types.NodeModeDPU || OvnKubeNode.Mode == types.NodeModeDPUHost {
-		if OvnKubeNode.MgmtPortNetdev == "" {
-			return fmt.Errorf("ovnkube-node-mgmt-port-netdev must be provided")
+		if OvnKubeNode.MgmtPortNetdev == "" && OvnKubeNode.MgmtPortDPResourceName == "" {
+			return fmt.Errorf("ovnkube-node-mgmt-port-netdev or ovnkube-node-mgmt-port-dp-resource-name must be provided")
```

Comment:
`ovnkube-node-mgmt-port-netdev` and `ovnkube-node-mgmt-port-dp-resource-name` are mutually exclusive. Ovnkube should either fail with error or throw a warning when both specified.

Reply:
I create a check and post a warning.